website/pages/std.md

```diff
@@ -3,30 +3,29 @@
 **This document is non-normative**.
 
 The VT6 protocol is structured into versioned **modules**.
-Before exchanging messages, servers and clients negotiate which VT6 modules, and versions thereof, they understand.
+Terminals and clients negotiate which VT6 modules, and versions thereof, they understand.
 
-Through the module structure, VT6 can adapt to clients and servers with vastly different feature sets.
+Through the module structure, VT6 can adapt to clients and terminals with vastly different feature sets.
 Through the use of versioning, new capabilities can be added and old functionality can be deprecated without having to break interoperability with old implementations.
 
-The navigation bar at the top of the screen shows all the modules that currently exist. For each module, there is a overview page, such as https://vt6.io/std/core/, that explains the module, its core features and how it fits together with other modules. Below this path, you can find the normative specifications for each module version, such as https://vt6.io/std/core/1.0/.
+The navigation bar at the top of the screen shows all the modules that currently exist. `Foundation` is an exception, since it defines fundamental protocols and interface contracts, and is not a versioned module that can be negotiated.
+For each module, there is an overview page, such as https://vt6.io/std/core/, that explains the module, its core features and how it fits together with other modules. Below this path, you can find the normative specifications for each module version, such as https://vt6.io/std/core/1.0/.
 
 ## Foundation
 
-Every VT6 server (that is, every terminal, proxy, etc.) needs to implement at least two modules:
+Every VT6 terminal (or proxy) needs to implement at least two modules:
```

Comment:
Since this document is nothing but a non-normative introduction, I think it suffices to mention proxies just once in the same place terminals are first mentioned.

Reply:
ok